index.bs

```diff
@@ -203,8 +230,27 @@ spec: mediaqueries-4; urlPrefix: https://drafts.csswg.org/mediaqueries-4/#
       </p>
 
       <p>
-        A <dfn>valid VideoConfiguration contentType</dfn> MUST include a media
-        type, subtype and full codecs description.
+        To check if a {{VideoConfiguration}} <var>configuration</var> is a
+        <dfn>valid video configuration</dfn>, the following steps MUST be run:
+        <ol>
+          <li>
+            If <var>configuration</var> is <code>null</code>, return
```

Comment:
VideoConfiguration is a dictionary, and a dictionary cannot be null. Rather, when a ES null is to be converted to an IDL dictionary, some fun things happen in https://heycam.github.io/webidl/#es-dictionary. If the dictionary has required members, it'll throw TypeError at step 5.

So, this step ought to be unreachable.